docs/gitrepo-add.md

```diff
@@ -206,15 +206,45 @@ After secret is created, specify the secret to `gitRepo.spec.helmSecretName`. Ma
 
 ### Use different helm credentials for each path
 
+Fleet allows you to define unique credentials for each Helm chart path in a Git repository using the `helmSecretNameForPaths` field.
+
 :::info
-`gitRepo.spec.helmSecretName` will be ignored if `gitRepo.spec.helmSecretNameForPaths` is provided
+If `gitRepo.spec.helmSecretNameForPaths` is defined, `gitRepo.spec.helmSecretName` is ignored.
 :::
 
-Create a file `secrets-path.yaml` that contains credentials for each path defined in a `GitRepo`. Credentials will not be used
-for paths that are not present in this file.
-The path is the actual path to the bundle (ie to a folder containing a `fleet.yaml` file) within the git repository, which might have more segments than the entry under `paths:`.
+Create a file named `secrets-path.yaml` that specifies credentials for each path in your `GitRepo`. The keys must match the full path to a bundle directory (a folder containing a `fleet.yaml file`), which may have more segments than the entry under `paths:`. If a path listed in the GitRepo is not included in this file, Fleet does not use credentials for it.
+
+:::note
+The file should be named `secrets-path.yaml`, otherwise it won't work.
+:::
+
+Example `GitRepo` resource:
+
+```yaml
+kind: GitRepo
+apiVersion: fleet.cattle.io/v1alpha1
+metadata:
+  name: gitrepo
+  namespace: fleet-local
+spec:
+  helmSecretNameForPaths: test-multipasswd
+  repo: https://github.com/0xavi0/fleet-examples
+  branch: helm-multi-passwd
+  paths:
+  - single-cluster/test-multipasswd/no-passwd
+  - single-cluster/test-multipasswd/passwd
```

Comment:
nit: the GitRepo could use a single path, to illustrate the fact that a path in the secret may have more segments than a path specified in the GitRepo:
```suggestion
  - single-cluster/test-multipasswd
```